app/assets/javascripts/application.js

```diff
@@ -19,14 +7,17 @@ jQuery(function($) {
   $('#payment-form').submit(function(event) {
     var $form = $(this);
     $form.find('button').prop('disabled', true);
-    Stripe.card.createToken($form, stripeResponseHandler);
-    return false;
+    if ($(".credit-card")[0].style.display != "none"){
```

Comment:
I just realized you're using `erb` templates and not `slim` which is completely cool but that example i wrote is in slim :P